plasma_framework/contracts/src/exits/payment/controllers/PaymentChallengeIFENotCanonical.sol

```diff
@@ -50,8 +51,10 @@ library PaymentChallengeIFENotCanonical {
         );
         require(address(condition) != address(0), "Spending condition contract not found");
 
+        // FIXME: move to the finalized interface as https://github.com/omisego/plasma-contracts/issues/214
+        // Also, the tests should verify the args correctness
         bool isSpentByInFlightTx = condition.verify(
-            ife.inputs[args.inFlightTxInputIndex].outputGuard,
+            bytes32(''), // tmp solution, we don't need outputGuard anymore for the interface of 214
```

Comment:
I was surprised seeing just a number without realizing its related to prev comment
```suggestion
            bytes32(''), // tmp solution, we don't need outputGuard anymore for the interface of :point-up: GH-214
```